onsp_co2/models/carbon_factor.py

```diff
@@ -72,6 +81,21 @@ def _compute_child_qty(self):
         for factor in self:
             factor.child_qty = len(factor.child_ids)
 
+    @api.depends("chart_of_account_ids")           
+    def _compute_chart_of_account_qty(self):
+        for factor in self:
+            factor.chart_of_account_qty = len(factor.chart_of_account_ids)
+            
+    @api.depends("product_ids")           
+    def _compute_product_qty(self):
+        for factor in self:
+            factor.product_qty = len(factor.product_ids)
+            
+    @api.depends("product_categ_ids")           
+    def _compute_product_categ_qty(self):
+        for factor in self:
+            factor.product_categ_qty = len(factor.product_categ_ids)
```

Comment:
Could you please use read_group as it is better performance wise?

You have 2 examples here: 
-  [OCA](https://github.com/OCA/project/blob/805cfc9366bc3ffb98ee90577e1738366ad4f09a/project_stock/models/project_task.py#L90-L96)
- [Odoo Enterprise](https://github.com/odoo/enterprise/blob/d0de5f1f5c6daa8537efacb62d73d2c8fd52c7f6/sign/models/res_partner.py#L10-L16)

Reply:
Fixed in the new commit